CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v0.5.0
```

Comment:
If we don't drop node v6, maybe it shouldn't be a major release?

Reply:
This is true, although I don't know what the next version will be, will rename to `next`.